tests/src/tilesetTests.cpp

```diff
@@ -0,0 +1,64 @@
+#include "tilesetTests.h"
+
+#include "testUtils.h"
+
+#include "cesium/omniverse/AssetRegistry.h"
+#include "cesium/omniverse/OmniTileset.h"
+#include "cesium/omniverse/UsdUtil.h"
+
+#include <carb/dictionary/DictionaryUtils.h>
+#include <carb/events/IEvents.h>
+#include <doctest/doctest.h>
+#include <omni/kit/IApp.h>
+
+pxr::SdfPath endToEndTilesetPath;
+bool endToEndTilesetLoaded = false;
+carb::events::ISubscriptionPtr endToEndTilesetSubscriptionPtr;
+class TilesetLoadListener;
+TilesetLoadListener* tilesetLoadListener;
```

Comment:
It felt weird not to use a smart pointer, but for a global I didn't think they made sense

Reply:
I would still switch to a `unique_ptr` just for internal consistency. In the cleanup function you can call `reset` on it to free the memory.